MODULE.bazel

```diff
@@ -306,6 +306,30 @@ gvm.graalvm(
 )
 use_repo(gvm, "graalvm_toolchains")
 
+http_file = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+# DO NOT UPDATE the jq version, this is used to verify compatibility with old versions.
```

Comment:
I started with `@yq_toolchains//:resolved_toolchain`, but then replaced it with this:
* `aspect_bazel_lib` presumably tracks latest `jq` whereas we would want to test and pin an oldest compatible version here
* having to add `use_extension`/`use_repo` to access `@yq_toolchains` makes it so that this pure test dependency would need to end up in Bazel's dist archive.